setup.py

```diff
@@ -51,7 +51,8 @@
         'ftw.profilehook',
         'Pillow',
         'ftw.colorbox',
-        'plone.api'
+        'plone.api',
+        'plone.batching>=1.0.4'
```

Comment:
We prefer to support older Plone versions without overriding the pinnings.
@shylux Please test it for older versions with an older `plone.batching` and report the problems here so that we can decide how to proceed.